addons/pos_self_order/static/src/app/self_order_service.js

```diff
@@ -187,7 +187,7 @@ export class SelfOrder extends Reactive {
         const availableCategories = this.productCategories
             .sort((a, b) => a.sequence - b.sequence)
             .filter((c) => prodByCategIds[c.id])
-            .sort((a, b) => categorySorter(a, b, this.config.iface_start_categ_id));
+            .sort((a, b) => categorySorter(a, b));
```

Comment:
```suggestion
            .sort((a, b) => a.sequence - b.sequence
};
```
You can then remove `categorySorter` completely